Yoast/yoast-components
# Full meta description editor clickable

## Summary

This PR can be summarized in the following changelog entry:

* All input wrappers in the `SnippetEditorFields` feel and behave like an input and move focus to the wrapped input.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Click the lower part of the meta description field and see whether it focusses the editor.

Fixes Yoast/wordpress-seo#9798
